packages/core/src/shared/utilities/cacheUtils.ts

```diff
@@ -69,16 +69,20 @@ export async function loadOr<T, K>(cache: KeyedCache<T, K>, key: K, fn: () => Pr
  * Transform functions are not invoked if the specified key does not exist in the cache.
  *
  * @param cache Target cache. The original is _not_ affected.
- * @param get Function applied to all **read** operations from the cache.
- * @param set Function applied to all **write** operations from the cache.
+ * @param loadTransform Function applied to all **read** operations from the cache.
+ * @param saveTransform Function applied to all **write** operations from the cache.
```

Comment:
"on" is the typical prefix for callbacks. it helps others to recognize patterns when we use consistent patterns
```suggestion
 * @param onLoad Function applied to all **read** operations from the cache.
 * @param onSave Function applied to all **write** operations from the cache.
```